### DISCUSSION TITLE
Aware of MusicKit?

### DISCUSSION THREAD

**Discussion by alexmyczko** [trunkmaster/nextspace#387] [Ideas]
I'm not sure how relevant this is/could be for NEXTSPACE, but are you aware of this?

https://github.com/leighsmith/MusicKit

and maybe these? https://github.com/schik?tab=repositories

**Comment by trunkmaster** [comment]
I know about MusicKit. It can be relevant to NEXTSPACE (fully ported to GNUstep and use SoundKit instead of SndKit). Now it's just port from NeXTSTEP to MacOS.
https://github.com/schik?tab=repositories has very interesting concepts/ideas which can be used for NEXTSPACE.

Thank you for information. Labeling this issue as "idea".
